Frameworks/UIKit/UIStoryboard.mm

```diff
@@ -51,6 +52,7 @@ + (instancetype)storyboardWithName:(NSString*)name bundle:(NSBundle*)bundle {
         if (storyInfo) {
             ret->_entryPoint = [storyInfo objectForKey:@"UIStoryboardDesignatedEntryPointIdentifier"];
             ret->_fileMap = [storyInfo objectForKey:@"UIViewControllerIdentifiersToNibNames"];
+            ret->_fileMapByStoryboardId = [storyInfo objectForKey:@"UIViewControllerStoryboardIdentifiersToNibNames"];
```

Comment:
Is the map stored in the nib under this key? We've maintained compatibility with reference platform nib files, we don't want to break it until we make a definitive decision to.